ui/ducks/send/send.js

```diff
@@ -1313,10 +1323,14 @@ const slice = createSlice({
         recipient.error = null;
         recipient.warning = null;
       } else {
```

Comment:
`isSendingToken` is removed because we want to show the user a warning regardless of what currency they are sending to a token contract